docs/en/samples/fuse_clean_policy.md

```diff
@@ -0,0 +1,110 @@
+# Demo - Set FUSE clean policy
+
+FUSE clean policy is set under the property `spec.fuse.cleanPolicy` of `Runtime`. There are 2 clean policies for FUSE Pod. `OnRuntimeDeleted` means that the FUSE Pod is cleaned only when the cache runtime is deleted, and `OnDemand` means that when the FUSE Pod is not used by any application Pod, the FUSE Pod will be cleaned.
+By default, the cleanup policy for FUSE Pod is `OnRuntimeDeleted` 
```

Comment:
If the cleanPolicy is `OnRuntimeDeleted`, how does it work?

Reply:
Do you mean add an demo of `OnRuntimeDeleted` showing how it workd?

Reply:
Yes， I want to make the demo complete.